sale_force_invoiced/view/sale_view.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should not be removed

Reply:
@mreficent This header is no longer required, assumed is UTF-8. 
See odoo documentation, there is the header is missing: 
https://www.odoo.com/documentation/11.0/reference/data.html
In the newer xml files in default Odoo you can also see that this line is absent.

Reply:
But your example is not valid. In https://www.odoo.com/documentation/10.0/reference/data.html the header is also missing.

Reply:
For being a correct xml with UTF-8 coded words, you should include that attribute and the header. If not, there can be problems depending on the lxml library implementation and version. What is not needed in python 3 is for python files.

Reply:
@mreficent I did not say it was new with Odoo 11.. 
In that case Odoo is also doing things wrong, for example in this file: https://github.com/odoo/odoo/blob/11.0/addons/hr_recruitment/views/hr_department_views.xml

(For extra confusion, odoo guidelines specify the utf-8 header for python files, but not for the xml files.. :) )